src/Test/L0/Worker/StepsRunnerL0.cs

```diff
@@ -326,11 +327,12 @@ private TestHostContext CreateTestContext([CallerMemberName] String testName = "
                     Assert.Equal(2, variableSet.Length);
                     variableSet[0].Verify(x => x.RunAsync());
                     variableSet[1].Verify(x => x.RunAsync(), Times.Never());
+                    Assert.Equal(TaskResult.Skipped, variableSet[1].Object.ExecutionContext.Result);
```

Comment:
rather than complicate the mock object with additional understanding and state management (i.e. the callback you added below that manipulates the state of the mock), i think `variableSet[1].Verify(x => x.Skip());` would be simpler here and consistent with the rest of the file.